src/latency.h

```diff
@@ -89,4 +91,34 @@ void latencyAddSample(const char *event, mstime_t latency);
 #define latencyRemoveNestedEvent(event_var,nested_var) \
     event_var += nested_var;
 
+struct durationStats {
+    long long cnt;
```

Comment:
It seems that they can't be negative.
```suggestion
    unsigned long long cnt;
```